release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -483,6 +483,20 @@ In {product-title} 4.12, support for Kuryr on clusters that run on {rh-openstack
 [id="ocp-4-13-removed-features"]
 === Removed features
 
+[id="ocp-4-13-z13-power8-x86v1-remove"]
+==== Specific hardware models on `ppc64le`, `s390x`, and `x86_64` v1 CPU architectures are removed
+
+In {product-title} 4.13, support for {op-system} functionality is removed for the following deprecated hardware models:
+
+* IBM POWER8 all models (ppc64le)
```

Comment:
Should this and the following be in backticks? 

```suggestion
* IBM POWER8 all models (`ppc64le`)
```

Reply:
Good idea 🥇